ledger/ledger-on-sql/src/test/suite/scala/com/daml/ledger/on/sql/ImmutableMigrationsSpec.scala

```diff
@@ -3,76 +3,10 @@
 
 package com.daml.ledger.on.sql
 
-import java.io.{BufferedReader, FileNotFoundException}
-import java.math.BigInteger
-import java.nio.charset.Charset
-import java.security.MessageDigest
-import java.util
+import com.daml.flyway.AbstractImmutableMigrationsSpec
 
-import com.daml.ledger.on.sql.ImmutableMigrationsSpec._
-import org.flywaydb.core.Flyway
-import org.flywaydb.core.api.configuration.FluentConfiguration
-import org.flywaydb.core.internal.resource.LoadableResource
-import org.flywaydb.core.internal.scanner.{LocationScannerCache, ResourceNameCache, Scanner}
-import org.scalatest.Matchers._
-import org.scalatest.WordSpec
-
-import scala.collection.JavaConverters._
-
-class ImmutableMigrationsSpec extends WordSpec {
-  "migration files" should {
-    "never change, according to their accompanying digest file" in {
-      val configuration = Flyway
-        .configure()
-        .locations(s"classpath:/$migrationsResourcePath")
-      val resourceScanner = flywayScanner(configuration)
-      val resources = resourceScanner.getResources("", ".sql").asScala.toSeq
-      resources.size should be >= 3
```

Comment:
I don't understand the purpose of the `>=` check here but have kept it alive in the refactored version. cc @SamirTalwar-DA